app/helpers/react_components/modals/welcome_insiders_modal.rb

```diff
@@ -0,0 +1,21 @@
+module ReactComponents
+  module Modals
+    class WelcomeInsidersModal < ReactComponent
```

Comment:
Maybe rename to:

```suggestion
    class WelcomeToInsidersModal < ReactComponent
```

Reply:
I feel it's a bit redundant. I'd prefer the classnames to match the slug, React component, and component key. what do you think @iHiD?

Reply:
> I feel it's a bit redundant

Well, we also have a `CopyToClipboardButton` component and not `CopyClipboardButton` 🤷 I won't die on this hill, but I like explicitness.

> I'd prefer the classnames to match the slug, React component, and component key.

Well, these could also be renamed? But feel free to ignore for now, it's low priority.